.eslintrc

```diff
@@ -10,6 +10,22 @@
   },
   "rules": {
     "@wordpress/dependency-group": "error",
-    "@wordpress/react-no-unsafe-timeout": "error"
+    "@wordpress/react-no-unsafe-timeout": "error",
+    "comma-dangle": [
+      "error",
+      {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
```

Comment:
This requires the trailing comma here:

```js
const foo = {
    one: 'baz',
    two: 'bar',
};
```

...but doesn't allow a trailing comma here:

```js
const foo = { one: 'baz', two: 'bar' };
```